tests/draft4/ref.json

```diff
@@ -208,5 +208,93 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "Recursive references between schemas",
+        "schema": {
+            "id": "http://localhost:1234/tree",
+            "description": "tree of nodes",
+            "type": "object",
+            "properties": {
+                "meta": {"type": "string"},
+                "nodes": {
+                    "type": "array",
+                    "items": {"$ref": "node"}
+                }
+            },
+            "required": ["meta", "nodes"],
+            "definitions": {
```

Comment:
How exactly does `{"$ref": "node"}` points to `http://localhost:1234/node` ? I don't follow.